rascaline/src/calculators/lode/radial_integral/spline.rs

```diff
@@ -63,6 +64,33 @@ impl LodeRadialIntegralSpline {
             center_contribution: radial_integral.compute_center_contribution()
         });
     }
+
+    pub fn from_tabulated(
+        parameters: LodeRadialIntegralSplineParameters,
+        spline_points: Vec<SplinePoint>
+    ) -> Result<LodeRadialIntegralSpline, Error> {
+
+        let spline_parameters = SplineParameters {
+            start: 0.0,
+            stop: parameters.cutoff,
+            shape: vec![parameters.max_angular + 1, parameters.max_radial],
+        };
+
+        let mut new_spline_points = Vec::new();
+        for spline_point in spline_points {
+            new_spline_points.push(
+                HermitSplinePoint{
+                    position: spline_point.position,
+                    values: spline_point.values.0.clone(),
+                    derivatives: spline_point.derivatives.0.clone(),
+                }
+            );
+        }
+
+        let spline = HermitCubicSpline::new(spline_parameters, new_spline_points);
+        return Ok(LodeRadialIntegralSpline{
+            spline: spline, center_contribution: Array1::zeros(parameters.max_radial)});
```

Comment:
`center_contribution` is currently set to zero here, we will also have to provide these values with the splines something like:

```python
radial_basis={"TabulatedRadialIntegral": {"points": spline_points, "center_contribution": xxx}},
```